wagtail/contrib/styleguide/wagtail_hooks.py

```diff
@@ -11,10 +11,10 @@
 @hooks.register('register_admin_urls')
 def register_admin_urls():
     return [
-        url(r'^styleguide/$', views.index, name='wagtailstyleguide'),
+        url(r'^styleguide/$', views.index, name='styleguide'),
```

Comment:
Yep, look for all the calls to ``reverse('styleguide')`` and change them there as well